ext/opentelemetry-ext-prometheus/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## Unreleased
+
```

Comment:
Are we leaving this blank?

Reply:
Is not blank it says unreleased, once is released changelogs will make sense

Reply:
@c24t are we doing changelogs on this fashion? I wonder if there's an easier way to manage these than attempting to update changelogs in all repos as we go along. Then again, this may play better with pypi.

Reply:
This is how we started the other changelogs. The idea is that each PR that makes a relevant change should add a line under "unreleased", and those lines get moved under a release-specific header when we cut the release.